backend/test/support/conn_case.ex

```diff
@@ -67,4 +67,12 @@ defmodule RichardBurtonWeb.ConnCase do
   def expect_mailer_send(n \\ 1) do
     expect(RichardBurton.MailerMock, :send, n, fn _ -> {:ok, "Whatever"} end)
   end
+
+  def mock_admin_logged_in(_context) do
+    RichardBurton.AuthMock
+    |> expect(:verify, fn _ -> {:ok, "admin-token"} end)
+    |> expect(:authorize, fn _, :admin -> :ok end)
+
+    %{}
+  end
```

Comment:
I think you can use `expect_auth_athorize_admin` defined above

Reply:
In its most recent version, the main difference y that I use `stub` instead of `expect` because I don't really care to test the amount of call to those auth methods